### ISSUE TITLE
[KB] kb.php?mode=cat > should provide an "edit" button

### ISSUE THREAD

**Issue by vendethiel** [MightyGorgon/icy_phoenix_plugins#11]
there's currently "delete" and "unapprove", I think "edit" makes sense

**NOTE:** this is super-minor thing, and since KB isn't really maintained...

**Comment by MWE001** [comment]
Actually, those two buttons kind of threw me off for a minute or two with some serious thoughts involved. 

 How can I decide to unapprove a article unless I click on it and read it? The same goes for delete. And if there was an edit button there, I wouldn't know if it needed edited unless I was on the article itself anyhow. 

So in my humblest of opinions, the unapprove and delete buttons needs to be on the bottom of the article next to the already there edit button. This is not a new thought to me. I thought this way back in the day on Integramod as well. I was going to move mine long ago but never did as the KB was never a big part of my site anyhow. Now it actually is needed. I will be using it. 

I can see overall why you would want the edit button there. if I can get one added I'll come back and post.

**Comment by vendethiel** [comment]
Mmmh... As I said it's really minor. It's more about consistency.

**Comment by MWE001** [comment]
yep, I can see that. I am hot on the trail now. Just got done with dinner and am working on that button.

**Comment by MWE001** [comment]
Ok So I can find all the code but this one is a bit over my head. I can't do it. Was worth a try. I know when to say enough is enough.
